lib/widgets/resources/motor.dart

```diff
@@ -42,20 +42,17 @@ class _ViamMotorWidgetState extends State<ViamMotorWidget> {
       });
       await Future.delayed(const Duration(milliseconds: 10));
```

Comment:
This was my fault, I should have left a comment originally explaining why we had this odd delay and a re attempt at stopping. Perhaps we include something like this?

```suggestion
      });
      // Sometimes the motor does not honor the first Stop call
      // So we wait a small amount of time and try again.
      await Future.delayed(const Duration(milliseconds: 10));
```